### DISCUSSION TITLE
April Fools Updates

### DISCUSSION THREAD

**Discussion by whatIsAUsernameidk** [PrismLauncher/PrismLauncher#1993] [Ideas]
I really like how the April Fools updates of Minecraft are there, but it would be nice if they had their own filter category. Furthermore I noticed that Java edition 2.0 isn't in the Version list, if it is compatible with this it would be nice if it was in the prism Launcher

**Comment by roke-julian-lockhart** [comment]
> https://github.com/orgs/PrismLauncher/discussions/1993#discussion-6018306

@whatIsAUsernameidk,

> I noticed that Java edition isn't in the Version list

...as opposed to what? Prism solely supports Java Edition. Did you mean Bedrock Edition?

**Reply by whatIsAUsernameidk** [reply]
I meant the old Java edition 2.0 April fools update, sorry. Editing to fix that now.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/orgs/PrismLauncher/discussions/1993#discussioncomment-9834175

@whatIsAUsernameidk, no need to apologize. I'd like to see them too.

**Reply by ItsZer0here** [reply]
> > https://github.com/orgs/PrismLauncher/discussions/1993#discussioncomment-9834175
> 
> @whatIsAUsernameidk, no need to apologize. I'd like to see them too.

me too!!!!!
